pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -290,10 +290,11 @@ sub _suggest_imunify360 {
 sub _suggest_iav {
     my ($self) = @_;
 
-    if ( !$self->{iav}{installed} ) {
+    if ( !$self->{iav}{installed} && !$self->{iavp}{licensed} ) {
         $self->_avplus_advice( action => 'installav', advice => 'bad' );
```

Comment:
It looks like this is changing to a fall-through, do we want to return here?

Reply:
sure that would work